.buildkite/commands/release-build.sh

```diff
@@ -8,3 +8,9 @@ bundle exec fastlane run configure_apply
 
 echo "--- :hammer_and_wrench: Building"
 bundle exec fastlane build_and_upload_release app:$1 skip_confirm:true skip_prechecks:true upload_to_play_store:true
+
+echo "--- 💾 Saving Artifact"
+for aab in build/*.aab; do
+  buildkite-agent artifact upload "$aab"
+  echo "<a href="artifact://$aab">$(basename "$aab")</a>" | buildkite-agent annotate --style info --context "beta-build-$aab"
```

Comment:
Should the context be "release-build" rather than "beta-build"?

Reply:
Good catch!

Though it doesn't really matter in practice because those annotations are attached to the pipeline and build so there's no risk of those annotations conflicting between beta and release pipelines… but better be consistent for readability of the pipeline 🙂

Reply:
```suggestion
  echo "<a href="artifact://$aab">$(basename "$aab")</a>" | buildkite-agent annotate --style info --context "release-build-$aab"
```